source/door_b_enb.rst

```diff
@@ -73,8 +73,6 @@ Our real-time web pages will alert us and the Lead System Engineer will call us.
 ...possibly useful things left over from the DEA-A poweroff page I cribbed from... TBR
 
 * Identify whether or not additional comm time is needed and if so ask the OC/LSE to request it.
-* Send an email to ``sot_red_alert@head`` and call a telecon with the FOT, SOT, and FDs to brief
```

Comment:
@richardjedgar was there a reason these lines got yanked? I suspect we talked about it but I don't remember.

Reply:
@richardjedgar ignore this comment, I see what happened now.